### DISCUSSION TITLE
How to use HTTP Proxy without install addon

### DISCUSSION THREAD

**Discussion by yangiak** [llamafilm/tesla-http-proxy-addon#141] [Q&A]
Hi, 

I wanted to use the HTTP proxy without an addon since my installation doesn't support addon feature. Is there a guide on configuring an HTTP proxy for using Fleet API?

Thanks 
YK

**Comment by iainbullock** [comment]
I have a docker version which might do what you need?

https://github.com/iainbullock/tesla-http-proxy-docker
